projects/ontimize-web-ngx-report/src/lib/components/report/o-report-viewer/o-report-viewer.component.ts

```diff
@@ -32,6 +38,13 @@ export class OReportViewerComponent {
       }
     );
   }
-
+  setFullscreenDialog(): void {
```

Comment:
I think it is better to move this method to a utils file because its used in other component too